Anchor.toml

```diff
@@ -3,13 +3,13 @@ seeds = false
 skip-lint = false
 
 [programs.localnet]
-sol_airdrop = "tXmC2ARKqzPoX6wQAVmDj25XAQUN6JQe8iz19QR5Lo3"
+dual_airdrop = "2fJcpdR6qzqDP7fBqvoJQ5PGYdaRFBNyUKZkZf5t12mr"
```

Comment:
Should we fork or maintain the code for that older version someplace? Or no need since its deployed to mainnet and will never change

Reply:
Program will never be changed and hopefully we can close it in a couple weeks. Would be a real mistake to leave it around unattended for the very long.
Code is always in the git history